scripts/Invoke-SasNetworkAwareField.ps1

```
@@ -99,6 +99,33 @@ function Test-SasAdOuShapeForNetworkTransition {
+
+function Test-SasCybernetShapeForNetworkTransition {
+    [CmdletBinding()]
+    param([string[]]$Arguments)
+
+    $values = @($Arguments)
+    if ($values.Count -lt 2) { return $false }
+    $mode = ([string]$values[0]).Trim().ToLowerInvariant()
+    if ($mode -eq 'canary') {
+        if ($values.Count -gt 6) { return $false }
+        foreach ($value in @($values | Select-Object -Skip 1)) {
+            if (-not (Test-SasCanaryTargetForNetworkTransition -Value ([string]$value)) { return $false }
```

PSScriptAnalyzer (1.25.0)

[error] Unexpected token '{' in expression or statement.

(UnexpectedToken)